### ISSUE TITLE
img for readme

### ISSUE THREAD

**Issue by sneha-belkhale** [sneha-belkhale/AI4Animation-js#1]
<img width="1056" alt="Screen Shot 2019-04-19 at 5 11 45 PM" src="https://user-images.githubusercontent.com/17795014/56417371-44dea780-62c6-11e9-8bcb-c01f109f7bdb.png">

**Comment by icodeforlove** [comment]
Any reason for the black on black. It's pretty tough to actually see what is going on.

**Comment by sneha-belkhale** [comment]
@icodeforlove yeah that was a stylistic choice -- I'm going to make a toggle for the demo to enable my debug mode which is much more informative. for now, I added extra lighting underneath and brightened the ground a bit 
<img width="1000" alt="lightImg" src="https://user-images.githubusercontent.com/17795014/56511517-5ee6e700-655f-11e9-9660-ca6d687ddd00.png">
